app/helpers/layout_helper.rb

```diff
@@ -98,6 +98,28 @@ def javascript(*args)
     content_for(:javascripts) { javascript_include_tag(*args) }
   end
 
+  def javascript_include_tag(*params, **kwargs)
+    # Workaround for overriding javasctipt load with webpack_asset_paths, should be removed when webpack_asset_paths is removed
+    if kwargs.is_a?(Hash) && kwargs[:source] == "webpack_asset_paths"
```

Comment:
kwargs should always be a `Hash`
```suggestion
    if kwargs[:source] == "webpack_asset_paths"
```